src/main.js

```diff
@@ -229,11 +229,17 @@ function saveThisPoster() {
 }
 //target is undefined
 //storing target value somewhere
-// function deleteMiniPoster(){
-//   var getId = event.target.id;
-//   for (var i = 0; i < savedPosters.length; i++) {
-//     if(savedPosters[i].id === getId){
-//       savedPosters.splice(i,1);
-//     }
-//   }
-// }
+function deleteMiniPoster(event){
+  // console.log(event);
+
+  var getId = parseInt(event.target.id);
```

Comment:
Great job knowing the distinction between your `==` and `===`! (Here are)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt] the `parseInt()` docs, which have some unique and unexpected use cases for this method.

Definitely a good one to have in your back pocket for when you need to type coerce to a number!